project/cmake/CMakeLists.txt

```diff
@@ -408,7 +408,13 @@ if(CMAKE_GENERATOR STREQUAL "Unix Makefiles")
 endif()
 
 # Prepare add-on build env
-foreach(binding ${addon_bindings})
+file(STRINGS ${CORE_SOURCE_DIR}/xbmc/addons/addon-bindings.mk bindings)
+string(REPLACE "\n" ";" bindings "${bindings}")
```

Comment:
No comment lines so no need for it. But you gave me an idea. Will update later today